tests/test_translation.h

```diff
@@ -145,6 +149,33 @@ TEST_CASE("[OptimizedTranslation] Generate from Translation and read messages")
 	CHECK(messages.size() == 0);
 }
 
+#ifdef TOOLS_ENABLED
+TEST_CASE("[Translation] CSV import") {
+	Ref<ResourceImporterCSVTranslation> import_csv_translation = memnew(ResourceImporterCSVTranslation);
+
+	Map<StringName, Variant> options;
+	options["compress"] = false;
+	options["delimiter"] = 0;
+
+	List<String> gen_files;
+
+	Error result = import_csv_translation->import(TestUtils::get_data_path("translations.csv"),
+			"", options, nullptr, &gen_files);
+	CHECK(result == OK);
+	CHECK(gen_files.size() == 2);
+
+	for (List<String>::Element *E = gen_files.front(); E; E = E->next()) {
```

Comment:
You can now iterate lists more easily:
```suggestion
	for (const String &file : gen_files) {
```

Reply:
Changed it